src/inference/src/dev/core_impl.cpp

```diff
@@ -418,7 +414,8 @@ ov::SoPtr<ov::ICompiledModel> ov::CoreImpl::import_model(std::istream& model,
                                                          const std::string& device_name,
                                                          const ov::AnyMap& config) const {
     auto parsed = parseDeviceNameIntoConfig(device_name, config);
-    auto exec = get_plugin(parsed._deviceName).import_model(model, config);
+    ov::AnyMap _config = property_manager.exclude_property(config);
```

Comment:
what if user passed enable_tbb_terminate / cache_dir here? It will be ignored without any exceptions

Reply:
I supposed we cannot pass core properties to plugin.
Anyhow, we should adopt the same actions for  `compile_model`, `query_model` and `import_model`

Reply:
> I supposed we cannot pass core properties to plugin

This is true.

But if I use:
```c++
ov::Core core;
core.import_model("blob.blob", "CPU", ov::terminate_tbb(true));
```
The option related to TBB will be ignored.

Reply:
`ov::terminate_tbb(true)` property is only visible on ov::core by `ov::set_property()`, but not pass to `core.import_model()`
Anyhow, do you think we should make core_properties is applied for each model or for each plugin?